### ISSUE TITLE
NVDA does not read aria-label in Outlook app

### ISSUE THREAD

**Issue by quidkid** [nvaccess/nvda#12513]
### Steps to reproduce:

1. Go to [https://putsmail.com/](url) and create a test email. Copy and paste the following into the body:
`<a href="https://www.google.com" aria-label="Google search">Bing search</a>`
and send it to your email.
2. Open Outlook desktop app on Windows.
2. With NVDA on, keyboard Tab to the link.
3. Notice that the screenreader does not announce the aria-label of "Google search" which is supposed to override the link name of "Bing search".

### Actual behavior:
Aria-labels are not recognized by NVDA.
### Expected behavior:
NVDA should read out the aria-label instead of the link text when present.
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
alpha-22774,44fabd74
#### Windows version:
Windows 10 Enterprise 10.0.18363
#### Name and version of other software in use when reproducing the issue:
Microsoft 365 Apps for Enterprise Outlook Version 2008 Build 13127.21506
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No
#### If add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by Adriani90** [comment]
@quidkid are you still having this issue with NVDA 2024.4.1? The test case you provided doesn't work anymore. Can you provide an updated link?
